baseapp/abci.go

```diff
@@ -196,7 +196,8 @@ func (app *BaseApp) BeginBlock(req abci.RequestBeginBlock) (res abci.ResponseBeg
 	app.deliverState.ctx = app.deliverState.ctx.
 		WithBlockGasMeter(gasMeter).
 		WithHeaderHash(req.Hash).
-		WithConsensusParams(app.GetConsensusParams(app.deliverState.ctx))
+		WithConsensusParams(app.GetConsensusParams(app.deliverState.ctx)).
+		WithVoteInfos(req.LastCommitInfo.Votes)
```

Comment:
I think we should use `GetVotes()` like in L217 just in case of nil pointer.

Reply:
You are right 👍